content/pt-br/docs/reference/scheduling/policies.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Políticas do escalonador
+content_type: concept
+sitemap:
+  priority: 0.2 # Scheduling priorities are deprecated
+weight: 30
+---
+
+<!-- overview -->
+
+Nas versões anteriores à v1.23 do Kubernetes, a política do escalonador pode ser usada para especificar os processos *predicates* e *priorities*. Por exemplo, você pode definir a política do escalonador rodando os seguintes comandos:
+```
+ `kube-scheduler --policy-config-file <filename>` 
+ ou
+  `kube-scheduler --policy-configmap <ConfigMap>`.
+ ```
+
+Essa política do escalonador não é compatível desde a v1.23 do Kubernetes. Flags associadas `policy-config-file`, `policy-configmap`, `policy-configmap-namespace` e `use-legacy-policy-config` também não são compatíveis. Em vez disso, use a API de [configuração do escalonador](/docs/reference/scheduling/config/) para conseguir um comportamento similar.
+
+## {{% heading "whatsnext" %}}
+
+* Descubra mais sobre o [escalonador](content/pt-br/docs/concepts/scheduling-eviction/kube-scheduler/)
```

Comment:
This link is currently broken. It should not be prefixed by `content`. Instead, it should start with `/pt-br/`. Example: https://github.com/kubernetes/website/blob/c44bdb2217adab9ce05f7b478f5cbd143b5640f2/content/pt-br/docs/concepts/configuration/secret.md?plain=1#L110